cmd2/cmd2.py

```diff
@@ -4813,7 +4813,12 @@ def _initialize_history(self, hist_file: str) -> None:
                           to this file when the application exits.
         """
         import json
-        import lzma
+        try:
+            import lzma
+            bz2 = None
```

Comment:
Overall your logic looks flawed.  

There appears to be no need for a `bz2` variable, but in this case you should set `lzma` to something, perhaps `True`.

Same comment applies to similar logic below.

In other case probably best to set witch to `False` instead of `None` to make this boolean and switch conditional logic as well.

Reply:
How about:

```
try:
    import lzma as compress_lib
except ModuleNotFoundError:  # pragma: no cover
    import bz2 as compress_lib
```

and then you can simply use `compress_lib.decompress()` unconditionally below?

Reply:
@osfrickler Yes I like that solution better, it is very clean.  Great suggestion.